mkdocs/docs/HPC/running_batch_jobs.md

```diff
@@ -803,6 +803,28 @@ The output of the various commands interacting with jobs (`qsub`,
 `stat`, ...) all depend on which `cluster` module is loaded.
 {% endif %}
 
+### cross cluster submission
+
+We made it possible to submit to a cluster different from the cluster that your current worker node is on. 
+This could come in handy if, for example, the submit command only works on the current clusters worker nodes, but the job itself can run on other clusters. An example of this is the `wsub` command of `worker`. More technical info why the `wsub` command can only be executed on the {{defaultcluster}} cluster can be found [here](troubleshooting.md#multi-job-submissions-on-a-non-default-cluster).
+
+```shell
+{% include "examples/Multi-job-submission/par_sweep/weather.pbs" %}
+```
+
+As mentioned before, the `wsub` command will only work on the `{{defaultcluster}}` cluster. But this does not mean that the job itself can't be executed on other clusters. For example if you want to submit to `{{othercluster}}` cluster, you can only change your submit cluster by using `module swap env/surm/{{othercluster}}` instead of `module swap cluster/{{othercluster}}`. The wsub command will then be executed on a {{defaultcluster}} worker node, but the jobs itself will be submitted to the `{{othercluster}}` cluster. It is ofcourse also possible to use an other cluster then `{{othercluster}}`.
```

Comment:
I would only refer to the `wsub` section once, and then keep the rest of this more general:

```suggestion
To submit jobs to the `{{othercluster}}` cluster, you can only change what is needed in your session environment to 
submit jobs to that particular cluster by using `module swap env/slurm/{{othercluster}}` instead of using `module swap cluster/{{othercluster}}`
(which also actives the modules that are installed specifically for {{othercluster}}, and which may not be compatible with the system you are working on).
That way, jobs that are submitted will be sent to the `{{othercluster}}` cluster. The same approach can be used to submit jobs to another cluster, of course.